scripts/register-parachain.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+source scripts/_init_var.sh
+
+RELAY_PORT=$((USER_PORT + 42))
+RELAY_INDEX=0
+BOOTNODES_ARGS=""
+
+# Will retrieve variable from the given network
+NETWORK=${NETWORK:-"alphanet"}
+WASM=$(eval echo "\$${NETWORK^^}_WASM")
+GENESIS=$(eval echo "\$${NETWORK^^}_GENESIS")
+TMP_FOLDER=$(eval echo "\$${NETWORK^^}_TMP_FOLDER")
+PARACHAIN_ID=$(eval echo "\$${NETWORK^^}_PARACHAIN_ID")
+
+if [ -z "$ROCOCO_SUDO_SEED" ]; then
+    echo "Missing \$ROCOCO_SUDO_SEED"
+    exit 1
+fi
+
+if [ ! -f "$WASM" ]; then
+    echo "Missing $WASM. Please run scripts/generate-parachain-specs.sh"
+    exit 1
+fi
+
+if [ ! -f "$GENESIS" ]; then
+    echo "Missing $GENESIS. Please run scripts/generate-parachain-specs.sh"
+    exit 1
+fi
+
+sha256sum $GENESIS $WASM
```

Comment:
is this doing anything? printing to STDOUT?

Reply:
Yes, just printing logs for comparing with ops